docs/src/starks/recap.md

```diff
@@ -242,49 +248,49 @@ The walkthrough above was for the fibonacci example which, because of its simpli
 
 $$
 H(x) = H_1(x^2) + x H_2(x^2)
 $$
 
-This way, we don't commit to $H$ but to $H_1$ and $H_2$. This is just an optimization at the code level; once again, the ideas remain exactly the same.
+This way, we don't commit to \\(H\\) but to \\(H_1\\) and \\(H_2\\). This is just an optimization at the code level; once again, the ideas remain exactly the same.
 
 ### FRI, low degree extensions and roots of unity
 
 We treated `FRI` as a black box entirely. However, there is one thing we do need to understand about it: low degree extensions.
 
-When applying `FRI` to a polynomial of degree $n$, we need to provide evaluations of it over a domain with *more* than $n$ points. In our case, the `DEEP` composition polynomial's degree is around the same as the trace's, which is, at most, $2^n - 1$ (because it interpolates the trace containing $2^n$ points).
+When applying `FRI` to a polynomial of degree \\(n\\), we need to provide evaluations of it over a domain with *more* than \\(n\\) points. In our case, the `DEEP` composition polynomial's degree is around the same as the trace's, which is, at most, \\(2^n - 1\\) (because it interpolates the trace containing \\(2^n\\) points).
 
-The domain we are going to choose to evaluate our `DEEP` polynomial on will be a set of *higher* roots of unity. In our fibonacci example, we will take a primitive $16$-th root of unity $\omega$. As a reminder, this means:
+The domain we are going to choose to evaluate our `DEEP` polynomial on will be a set of *higher* roots of unity. In our fibonacci example, we will take a primitive \\(16\\)-th root of unity \\(\omega\\). As a reminder, this means:
 
-- $\omega$ is an $16$-th root of unity, i.e., $\omega^{16} = 1$.
-- Every $16$-th root of unity is of the form $\omega^i$ for some $0 \leq i \leq 15$.
+- \\(\omega\\) is an \\(16\\)-th root of unity, i.e., \\(\omega^{16} = 1\\).
+- Every \\(16\\)-th root of unity is of the form \\(\omega^i\\) for some \\(0 \leq i \leq 15\\).
 
-Additionally, we also take it so that $\omega$ satisfies $\omega^2 = g$ ($g$ being the $8$-th primitive root of unity we used to construct `t`).
+Additionally, we also take it so that \\(\omega\\) satisfies \\(\omega^2 = g\\) (\\(g\\) being the \\(8\\)-th primitive root of unity we used to construct `t`).
 
-The evaluation of $t$ on the set $\{\omega^i : 0 \leq i \leq 15\}$ is called a *low degree extension* (`LDE`) of $t$. Notice this is not a new polynomial, they're evaluations of $t$ on some set of points. Also note that, because $\omega^2 = g$, the `LDE` contains all the evaluations of $t$ on the set of powers of $g$. In fact,
+The evaluation of $t$ on the set \\(\{\omega^i : 0 \leq i \leq 15\}\\) is called a *low degree extension* (`LDE`) of \\(t\\). Notice this is not a new polynomial, they're evaluations of \\(t\\) on some set of points. Also note that, because \\(\omega^2 = g\\), the `LDE` contains all the evaluations of \\(t\\) on the set of powers of \\(g\\). In fact,
```

Comment:
```suggestion
The evaluation of \\(t\\) on the set \\(\{\omega^i : 0 \leq i \leq 15\}\\) is called a *low degree extension* (`LDE`) of \\(t\\). Notice this is not a new polynomial, they're evaluations of \\(t\\) on some set of points. Also note that, because \\(\omega^2 = g\\), the `LDE` contains all the evaluations of \\(t\\) on the set of powers of \\(g\\). In fact,
```